### ISSUE TITLE
"export default from" lint an error

### ISSUE THREAD

**Issue by HypnosNova** [babel/babel-eslint#731]
I use @babel/plugin-proposal-export-default-from(version 7.2.0) plugin so that I can write:
export XXXXX from './xxxxx';   （instead of writing：export {XXXXX} from './xxxxx' and in xxxxx.js use export default XXXXX instead of use export {XXXXX}）
But eslint shows the gramma mistake and auto fix the code to:
exportXXXXXfrom './xxxxx;

I use rollup and babel to build the whole project and everything is correct, just the eslint shows the gramma error. How to solve this problem. Is it a bug in babel-eslint?

The error says： [eslint] There should be no space after 'export'.

**Comment by johnmutuma5** [comment]
Also experiencing this. Any solutions? @HypnosNova

**Comment by appsparkler** [comment]
This error was caused by `eslint-config-google`.  I updated my rules with the following and the errors stopped:

```javascript
  "object-curly-spacing": 0,
    "comma-dangle": ["error", {
      exports: "never",
      imports: "never",
      arrays: "always-multiline",
      objects: "always-multiline",
    }]
```

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [`@babel/eslint-parser` has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
